backend/src/main/java/com/bakdata/conquery/models/identifiable/IdMap.java

```diff
@@ -10,11 +9,19 @@
 import java.util.concurrent.ConcurrentMap;
 import java.util.stream.Stream;
 
+import javax.validation.Valid;
+
 import com.bakdata.conquery.models.identifiable.ids.IId;
 import com.google.common.collect.ForwardingMap;
 
-public class IdMap<ID extends IId<? super V>, V extends Identifiable<? extends ID>> extends ForwardingMap <ID ,V > implements Iterable<V>{
+/**
+ * A typesafe mapping for the ID-system that does not allow a remapping of an existing key.
+ * 
+ * @implNote implementation of {@link Iterable} is dropped, because hibernate could not decide on how to validate this map (either with an map-extractor or an iterable-extractor).
```

Comment:
Freches Hibernate

Reply:
Gäbe es da nicht ne Anno die Hibernate sagt wie man da Validaten soll

Reply:
https://stackoverflow.com/a/15935133/409761

Hilft das?

Reply:
Das ist ein anderes Problem. So wie ich das verstehe, weiß Hibernate da, wie er die Collection auspacken soll, nur nicht was drinne ist. Hier wusste Hibernate nicht, wie es auspacken soll.